internal/controllers/machineset/machineset_controller.go

```diff
@@ -880,8 +885,7 @@ func (r *Reconciler) updateStatus(ctx context.Context, cluster *clusterv1.Cluste
 
 		node, err := r.getMachineNode(ctx, cluster, machine)
 		if err != nil && machine.GetDeletionTimestamp().IsZero() {
-			log.Error(err, "Unable to retrieve Node status", "node", klog.KObj(node))
-			continue
+			return errors.Wrapf(err, "Unable to retrieve the status of Node %s", klog.KObj(node))
```

Comment:
```suggestion
			return errors.Wrapf(err, "unable to retrieve the status of Node %s", klog.KObj(node))
```

opportunity to lowercase this error.